worker/src/durable.rs

```diff
@@ -495,6 +564,46 @@ impl<'a> ListOptions<'a> {
     }
 }
 
+pub struct ScheduledTime {
```

Comment:
Nitpick but this struct should probably impl `From<Duration>`. And a small explanation for why this exists in some documentation would probably be useful as it isn't exactly clear from the usage in `set_alarm`.